packages/@ember/-internals/routing/lib/services/router.ts

```diff
@@ -511,7 +510,10 @@ RouterService.reopen(Evented, {
      @type String
      @public
    */
-  currentRouteName: readOnly('_router.currentRouteName'),
```

Comment:
why were these still using readOnly?

Reply:
Likely because no one had updated them.

However, we should test to confirm that this has the same semantics: I *believe* people depending on these intermediate getters with classic computed properties (`@computed` or any of the CP macros) will not get the expected reactivity behavior unless these are decorated with `@dependentKeyCompat`.

Reply:
ah, so would it be safe to say that _until_ `@computed` is deprecated _and_ removed, all getters in the framework must use `@dependentKeyCompat`?

Reply:
Minimally, that is true for anything which should be 'reactive'.